pkg/controller/common/manifestprocessing.go

```diff
@@ -61,8 +62,7 @@ func (p *ManifestProcessor) ProcessManifests(manifests []manifest.Manifest, comp
 				allErrors = append(allErrors, err)
 				continue
 			}
-			obj := &unstructured.Unstructured{}
-			_, _, err = unstructured.UnstructuredJSONScheme.Decode(rawJSON, nil, obj)
+			obj, err := runtime.Decode(p.JSONSerializer, rawJSON)
```

Comment:
This should be changed to use 'unstructured.JSONFallbackEncoder'.  That should fix the issues with unregistered resources in the scheme.

Reply:
Whoops, scratch that, that's only for Encoding.